ftdetect/helm.vim

```diff
@@ -1,4 +1,14 @@
-autocmd BufRead,BufNewFile */templates/*.{yaml,yml},*/templates/*.tpl,*.gotmpl,helmfile*.{yaml,yml} set ft=helm
+function! s:isHelm()
+  let filepath = expand("%:p")
+  let filename = expand("%:t")
+  if filepath =~ '\v/(templates|charts)/.*\.(ya?ml|gotmpl|tpl|txt)$' | return 1 | en
+  if filename =~ '\v(helmfile).ya?ml' | return 1 | en
+  if getline(1) =~ '^apiVersion:' || getline(2) =~ '^apiVersion:' | return 1 | en
```

Comment:
This would also set the filetype of normal kubernetes yaml files to helm. I'm not sure if this would be a problem.

Reply:
Maybe I could add a condition and only do this if the helm language server is installed. What do you think?

Reply:
I guess it's not really needed, I was just folloing an idea from #7 , I've removed it.